### PR TITLE
fix doc links

### PR DESCRIPTION
**Pull request by gnavvy**
Shall we keep these doc updates in the 5.0-release branch or in master?

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ To make this simple concept work, deck.gl handles a number of challenges:
 
 ## Ecosystem
 
-deck.gl is one of the main frameworks in the [vis.gl](https://vis.gl) framework suite.
+deck.gl is one of the main frameworks in the [vis.gl](http://vis.gl) framework suite.
```
Comment (ibgreen) [inline docs/README.md]: If vis.gl doesn't support https we should file a bug on that repo @jckr @balthazar

File: docs/advanced/picking.md [hunk 2]
```diff
@@ -40,34 +40,32 @@ To take full control of picking, a layer need to take the following steps:
...
-  const {value, size} = attribute;
-
-  let i = 0;
-  for (const object of data) {
-    const pickingColor = this.encodePickingColor(i);
-    value[i++] = pickingColor[0];
-    value[i++] = pickingColor[1];
-    value[i++] = pickingColor[2];
-  }
+    const {data} = this.props;
```
Comment (ibgreen) [inline docs/advanced/picking.md]: Don't we want to use spaces instead of tabs?

File: docs/whats-new.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ Release date: 21, Dec 2017
 
 All new additions to the official deck.gl 5.0 API are listed here. Note that in addition to the official new features in this release, deck.gl 5.0 also contains a number of significant under the hoods changes to prepare for new features and optimizations. Some of these are available as experimental APIs, see below.
 
-As always, for information on deprecations and how to update your code in response to any API changes, please read the deck.gl [Upgrade Guide](/docs/upgrade-guide.md).
+As always, for information on deprecations and how to update your code in response to any API changes, please read the deck.gl [upgrade Guide](/docs/upgrade-guide.md).
```
Comment (ibgreen) [inline docs/whats-new.md]: Revert this change, chapter titles should be Capitalized.

**Comment by Pessimistress** [comment]
The layer source code links in API reference are also broken.

File: docs/README.md (hunk 1 above)
Comment (balthazar) [reply docs/README.md]: Custom domains hosted on github pages do not support SSL

**Comment by ibgreen** [comment]
> Shall we keep these doc updates in the 5.0-release branch or in master?

Per process, this should go into master and be cherry-picked into 5.0-release. We could do it the other way but always better to follow process, less surprises.

File: docs/README.md (hunk 1 above)
Comment (ibgreen) [reply docs/README.md]: > Custom domains hosted on github pages do not support SSL

Hmm, I thought it worked for our other websites but seems it doesn't. Pity...
